site.yml

```
@@ -64,7 +64,7 @@ asciidoc:
     php-supported-versions-url: https://secure.php.net/supported-versions.php
     http-status-codes-base-url: https://developer.mozilla.org/en-US/docs/Web/Status
     minimum-php-version: 7.1
```

Do you also need to do:
```suggestion
    minimum-php-version: '7.1'
```
similar to #2453